### PR TITLE
Add usage, known limitations and performance sections

### PR DESCRIPTION
**Pull request by jaap3**

### CODE REVIEW

**Comment by jaap3** [comment]
Ref #3

File: README.rst [hunk 1]
```diff
@@ -50,6 +51,44 @@ use this query::
...
+  for database fields, not methods.
+
+.. _ATOMIC_REBUILD: http://docs.wagtail.io/en/v1.9/topics/search/backends.html#atomic-rebuild
+
+
+Performance
+~~~~~~~~~~~
+
+The PostgreSQL search backend has been tried and tested on a few small
+to medium sized website and it's performance compares favorably to that
```
Comment (BertrandBordage) [inline README.rst]: "**its** performance"

File: README.rst [hunk 2]
```diff
@@ -50,6 +51,44 @@ use this query::
...
+of ElasticSearch.
+
+Some noticable speed improvments are in place when using PostgreSQL >= 9.5.
```
Comment (BertrandBordage) [inline README.rst]: "improvements"

**Review by BertrandBordage — CHANGES_REQUESTED** [review]
Excellent, thanks! I didn’t know about `ATOMIC_REBUILD`, I’ll have to check that.

**Comment by BertrandBordage** [comment]
Hum, `ATOMIC_REBUILD` will be easy to implement with PostgreSQL :)

**Pull request merged**